camel/typing.py

```diff
@@ -24,10 +24,13 @@ class RoleType(Enum):
 
 
 class ModelType(Enum):
-    GPT_3_5_TURBO = "gpt-3.5-turbo"
-    GPT_3_5_TURBO_16K = "gpt-3.5-turbo-16k"
     GPT_4 = "gpt-4"
     GPT_4_32k = "gpt-4-32k"
+    GPT_4_TURBO = "gpt-4-1106-preview"
+    GPT_4_TURBO_VISION = "gpt-4-vision-preview"
+    GPT_3_5_TURBO = "gpt-3.5-turbo-1106"
+    GPT_3_5_TURBO_16k = "gpt-3.5-turbo-1106"
```

Comment:
Prefer moving the `GPT_3_5_TURBO` and `GPT_3_5_TURBO_16k` to the top.

Reply:
> Prefer moving the `GPT_3_5_TURBO` and `GPT_3_5_TURBO_16k` to the top.

done